libraries/core-react/src/components/Menu/Menu.tsx

```diff
@@ -64,6 +64,13 @@ const MenuContainer = forwardRef<HTMLDivElement, MenuContainerProps>(
     useGlobalKeyPress('Escape', () => {
       if (open && onClose !== null) {
         onClose()
+        anchorEl.focus()
+      }
+    })
+
+    useGlobalKeyPress('Enter', () => {
+      if (open && onClose !== null) {
+        if (window.document.contains(anchorEl)) anchorEl.focus()
```

Comment:
Should we also call the `onClose()` here?

Reply:
This is handled by MenuItem

Reply:
Ah, because its a `button` now right 🙈